lldb/test/API/python_api/formatters/TestFormattersSBAPI.py

```diff
@@ -155,6 +155,18 @@ def cleanup():
             ],
         )
```

Comment:
```suggestion
        )
        self.dbg.GetCategory("CCCSynth").SetEnabled(False)
```

I'm not sure how lldb chooses the summary providers in case of conflicts, but since that's not what we're testing here, we might as well remove the ambiguity.

Reply:
I don't think we can have a conflict. In this test we have several categories which are enabled one after the other. I just add a new one (with a new name CCCSynth2).

Reply:
You have two categories, but they both have a summary provider for the same type (`CCC`). (Enabling one category does not automatically disable the other ones.) Right now, lldb seems to pick the one you want, but I don't think we promise that anywhere (in fact, I wouldn't be surprised if this comes down to some (nondeterministic) ordering in some hash container). By disabling the first category, we make sure that lldb always pick the one we want (in case, e.g., the container ordering changes). Alternatively, you could just attach the summary provider to a different type (CCC2?)